client/src/pages/course/score.tsx

```diff
@@ -145,14 +177,77 @@ export function Page(props: CoursePageProps) {
             <Text mark>Total score and position is updated every day at 04:00 GMT+3</Text>
             {renderCsvExportButton(props, handleLoadCsv)}
           </Row>
-          {renderTable(loaded, students.content, columns, students.pagination, getCourseScore, cityName, mentor)}
+          {renderTable(
+            loaded,
+            students.content,
+            getVisibleColumns(columns),
+            students.pagination,
+            getCourseScore,
+            cityName,
+            mentor,
+            handleSettings,
+          )}
         </Spin>
+        {renderSettingsModal(getVisibleTasks(courseTasks), isVisibleSetting, handleModalCancel, handleModalOk)}
       </Layout.Content>
       <style jsx>{styles}</style>
     </>
   );
 }
 
+function renderSettingsModal(
+  courseTasks: any[],
```

Comment:
I remember that we have some interface for CourseTask

Reply:
done